### PR TITLE
User/admin home

### PR DESCRIPTION
**Pull request by kenichi127**
## Description / Purpose 
It is a homepage of the user and the manager
Search, add data (admin only), send food list, send mail, search shelters
Fixes issue:https://docs.google.com/spreadsheets/d/1BHy8K_EvpG4fBLjujxShiZWe2-tNs0Jj/edit#gid=905621989
task:6
# # To test this PR
(all)
1.Create two user
2. The first one created is the administrator, the second and subsequent ones become the users.
(Search)
1. Select the desired date, time and place.
2. Click Search button.
3. The answer page will appear
(Add data (administrator only))
1. Click on the add button
2. A page for adding will appear
(Contribution)
1.Click the button of contribution.
2.Contribution page will appear.
(Food list)
1. Click the button of the food list.
2. Food list page comes out
(send the mail)
1. Click the Send Mail button
2. Email will be sent. (Return to home)
(shelter search)
1. Click on the shelter button
2. Search for shelters on Googlemap
** Command **
php artisan migrate:refresh

 **screenshot**
(admin home)
image
(additional page)
image
(user home)
image
(Search page)
image
(foodlist page)
image
(Contribution Page)
image
(shelter searching)
image

### CODE REVIEW

File: app/Http/Controllers/SampleController.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use Illuminate\Support\Facades\Mail;
+use App\Mail\SampleNotification;
+
+class SampleController extends Controller
```
Comment (AubreyV) [inline app/Http/Controllers/SampleController.php]: Please avoid using non informative class name such us SampleController, TestController and etc.

File: app/Http/Controllers/SampleController.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+{
+  public function SampleNotification()
+  {
+    $name = auth()->user()->name;
+    $text = auth()->user()->name;
+    $to = auth()->user()->D_email;
+
+    Mail::to($to)->send(new SampleNotification($name, $text));
```
Comment (AubreyV) [inline app/Http/Controllers/SampleController.php]: Please change the name of your SampleNotification, use something that describe what it does, like EmailNotification or something else.

File: resources/views/emails/sample_notification.blade.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```
Comment (AubreyV) [inline resources/views/emails/sample_notification.blade.php]: Please change the name of this blade file to something more understandable, like, email notification.

File: resources/views/layouts/user.blade.php [hunk 4]
```diff
@@ -32,7 +32,26 @@
                 <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="{{ __('Toggle navigation') }}">
                     <span class="navbar-toggler-icon"></span>
                 </button>
+<<<<<<< Updated upstream
```
Comment (AubreyV) [inline resources/views/layouts/user.blade.php]: Please delete this. This is a git conflict, refrain from pushing git conflicts.

File: resources/views/layouts/user.blade.php [hunk 5]
```diff
@@ -32,7 +32,26 @@
...
+                <h3 class="bg-success">Today is $data%.</h3>
+                @elseif($data>=20)
+                <h3 class="bg-info">Today is $data%.</h3>
+                @elseif($data>=0)
+                <h3 class="bg-light">Today is $data%.</h3>
+                @else($data==null)
+                <h3>I am sorry  No data.</h3>
+                @endif
+                </div>
+>>>>>>> Stashed changes
```
Comment (AubreyV) [inline resources/views/layouts/user.blade.php]: this one also

File: app/Mail/SampleNotification.php [hunk 6]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class SampleNotification extends Mailable
```
Comment (AubreyV) [inline app/Mail/SampleNotification.php]: This is still SampleNotification, you should change to a readable and understandable file name and class name.

**Review by AubreyV — APPROVED** [review]

**Comment by hayato0309** [comment]
When user tries to register, this error is displayed.
https://gyazo.com/c9c8f4d19501cd2c6f5d2b52c83bd3de

**Review by hayato0309 — APPROVED** [review]
Approved PR#3
1. After clicking buttons, the pages are displayed properly.

**Pull request merged**